qemu/tests/cfg/virtio_scsi_cdrom.cfg

```diff
@@ -64,6 +64,32 @@
                     image_name_stg24 = images/storage24
                     image_name_stg25 = images/storage25
                     image_size_equal = 1G
+                    image_size_extra_stg = 1G
```

Comment:
If we can use the param image_size_equal directly to replace new param image_size_extra_stg? it seems this param isn't used before~

Reply:
Updated